source/includes/options-mongodrdl.yaml

```diff
@@ -270,5 +270,36 @@ description: |
 
   Specifies the database where the schema information is stored. This
   option is not used by the :commandoption:`sample <sample>` command.
+---
+program: mongodrdl
+name: uri
+directive: option
+args: <mongodb-uri>
+description: |
+  .. versionadded:: 2.12
 
+  MongoDB URI connection string.
+optional: true
+---
+program: mongodrdl
+name: gssapiHostName
+default: remote server's address
+directive: option
+args: <hostname>
+description: |
+  .. versionadded:: 2.12
```

Comment:
the `gssapiHostName` and `gssapiServiceName` options have existed since at least 2016, as far as I can tell from the `mongodrdl` git history! Not sure if this fact is worth changing this `versionadded` field to something earlier (or omitting it entirely if possible).

Reply:
Whoops! Took out the "New in 2.12" line.